### PR TITLE
Fix shout sounds not being initialized from the default theme when current theme lacks them

### LINKED ISSUE #283: Default Objection/Hold It/Take That shout SFX are not initialized if theme lacks them
The animations for interjections are properly grabbed, but the SFX are missing. This means that the client properly attempts to find the sound effect in current theme, but does not search the default theme if it fails.

### PR DESCRIPTION
**Pull request by in1tiate**
Fixes #283

### CODE REVIEW

**Review by Crystalwarrior — APPROVED** [review]
Simple and effective.

File: src/aosfxplayer.cpp [hunk 1]
```diff
@@ -37,11 +37,16 @@ void AOSfxPlayer::play(QString p_sfx, QString p_char, QString shout,
...
   QString sound_path = ao_app->get_sfx_suffix(ao_app->get_sounds_path(p_sfx));
 
-  if (shout != "")
+  if (shout != "") {
     misc_path = ao_app->get_sfx_suffix(ao_app->get_base_path() + "misc/" +
                                        shout + "/" + p_sfx);
+    theme_path = ao_app->get_sfx_suffix(ao_app->get_theme_path(p_sfx));
+    if (!file_exists(theme_path))
+      theme_path = ao_app->get_sfx_suffix(ao_app->get_default_theme_path(p_sfx));
+  }
```
Comment (oldmud0) [inline src/aosfxplayer.cpp:49]: So did it never check the theme in the first place?

**Review by oldmud0 — COMMENTED** [review]
What a mess.

File: src/aosfxplayer.cpp (hunk 1 above)
Comment (in1tiate) [reply src/aosfxplayer.cpp:49]: Seems like it.

**Review by oldmud0 — APPROVED** [review]

**Pull request merged**